README.md

```diff
@@ -176,26 +196,23 @@ and upgrading to a sufficiently recent compiler version is recommended in this c
 NCTools has been built and tested with GCC and Intel compilers.
 
 ## Documentation
-
-An embarrassingly small number of the commands have man pages.  These man pages
-are written using [AsciiDoc](http://asciidoc.org/) format.  If AsciiDoc is found
-on the system, the man pages will be built and installed automatically.
-
-Most commands support either a `-h` or `--help` options.  In some cases, this
-should be enough to get you started using the commands.  We are in the process
-of collecting more information on the tools available in this package.  These
-will be added when available.
-
-Commonly, to combine restart and history files, one should run is-compressed on
-the candidate file first.
-
-If it returns 0, it's land compressed file that needs:
-1. combine-ncc inputfile1 inputfile2 outputfile
-2. decompress-ncc inputfile outputfile
-
-If is-compressed returns non-zero, it's a regular distributed file that gets:
-1. mppnccombine outputfile inputfile1 inputfile2 ...
-
+Documentation for on using individual applications (tools) may be obtained by running
```

Comment:
"Documentation for on" wording is awkward.

Reply:
fixed